documentation/staging/content/userguide/managing-domains/model-in-image/auxiliary-images.md

```diff
@@ -18,6 +18,8 @@ description = "Auxiliary images are an alternative approach for supplying a doma
      - [Configuration example 1: Basic configuration](#configuration-example-1-basic-configuration)
      - [Configuration example 2: Source locations](#configuration-example-2-source-locations)
      - [Configuration example 3: Multiple images](#configuration-example-3-multiple-images)
+ - [Automated upgrade of the weblogic.oracle/v8 schema auxiliary images configuration](#automated-upgrade-of-the-weblogicoraclev8-schema-auxiliary-images-configuration)
+ - [Domain upgrade tool to manually upgrade the weblogic.oracle/v8 schema domain resource](#domain-upgrade-tool-to-manually-upgrade-the-weblogicoraclev8-schema-domain-resource)
```

Comment:
I could be wrong, but the TOC doesn't look like it includes all of the new sections.

Reply:
Updated the TOC to include sample sub-sections.